website/docs/d/appid_token_config.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+subcategory: "AppID Management"
+layout: "ibm"
+page_title: "IBM: AppID Token Configuration"
+description: |-
+Retrieves AppID Token Configuration.
+---
+
+# ibm_appid_token_config
+Retrieve information about an IBM Cloud AppID Management Services token configuration. For more information, refer to [Customizing AppID tokens](https://cloud.ibm.com/docs/appid?topic=appid-customizing-tokens).
+
+## Example usage
+
+```terraform
+data "ibm_appid_token_config" "tc" {
+    tenant_id = var.tenant_id
+}
+```
+
+## Argument reference
+Review the argument references that you can specify for your data source.
+
+- `tenant_id` - (String) The AppID instance GUID
```

Comment:
For arguments section can you add its required/optional
 eg tenant_id` - (Requires, String) The AppID instance GUID

Reply:
I was wondering if you considered using https://github.com/hashicorp/terraform-plugin-docs in the future? I know this is alpha at the moment, but I was already getting decent results for unoficial AppID provider :) anyway, I will add these manual changes as requested